packages/trpc/server/routers/viewer/eventTypes/getEventTypesFromGroup.handler.ts

```diff
@@ -46,6 +50,7 @@ export const getEventTypesFromGroup = async ({ ctx, input }: GetByViewerOptions)
         {
           where: {
             teamId: null,
+            schedulingType: null,
```

Comment:
Where will the Managed Event eventtypes show up?

Reply:
Managed event type would still show up in the user personal event tab. When you create a managed event type it is created on team level with scheduling type "MANAGED" and on user level with scheduling type null

Reply:
We were earlier filtering this after prisma query to prevent displaying two event types